### PR TITLE
Add the Allow Zero GrandTotal field to the Sales page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds the Allow Zero GrandTotal field to the Sales page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/configuration/sales/sales.html

### CODE REVIEW

File: src/configuration/sales/sales.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
 |--- |--- |--- |
 |Allow Reorder|Store View|Determines if the customers can reorder from their accounts. Options: Yes / No|
 
+## Allow Zero GrandTotal
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
```
Comment (dobooth) [inline src/configuration/sales/sales.md:41]: "Grand Total" and "Credit memo" are both two words.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/configuration/sales/sales.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/configuration/sales/sales.md:41]: Hi, @dobooth 
Done! thank you😊

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1191`.

**Pull request merged**